config.py

```diff
@@ -30,43 +30,52 @@
 meg_dir = os.path.join(study_path, 'MEG')
 
 ###############################################################################
-# DEFINE SUBJECTS
+# SUBJECTS / RUNS
 # ---------------
 #
-# The MEG-data needs to be stored in a folder named study_path/MEG/my_subject/
-
-base_raw_fname = '{subject}_audvis{run}_raw.fif'
-
-# How did you name your runs?
-# naming should be consistant over subjects
-# put the number of runs you ideally expect to have per subject
-# the scripts will issue a warning if there are less
-# leave empty if there is just one file
-runs =  [''] # ['run01', 'run02']
+# The MEG-data need to be stored in a folder
+# named my_study_path/MEG/my_subject/
 
+# This is the name of your experimnet
+study_name = 'audvis'
```

Comment:
Where do you use it?

Reply:
`base_raw_fname = '{subject}_' + study_name + '{run}_raw.fif'`